AirCasting/MobilePeripheralSessionManager.swift

```diff
@@ -0,0 +1,73 @@
+// Created by Lunar on 07/06/2021.
+//
+
+import Foundation
+import CoreBluetooth
+
+class MobilePeripheralSessionManager {
+    private let measurementStreamStorage: MeasurementStreamStorage
+    private lazy var locationProvider = LocationProvider()
+    
+    var activeMobileSession: MobileSession?
```

Comment:
`private(set)` if there is a reason to call a specific name to set this session the property setter should not be available to be called outside of the scope.